nengo/params.py

```diff
@@ -163,6 +163,12 @@ def validate(self, instance, value):
     def raise_error(self):
         raise ObsoleteError(self.short_msg, since=self.since, url=self.url)
 
+    def hashvalue(self, instance):
+        return Unconfigurable  # otherwise __get__ throws an error
+
+    def equal(self, instance_a, instance_b):
+        return True  # otherwise __get__ throws an error
+
```

Comment:
Just wondering out loud whether there's a better way to do `__get__` that avoids having to do this. Also, I would bust these out into a separate commit since they're not directly related to the main change.